pulsar/tests/common.py

```diff
@@ -0,0 +1,97 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.dev import get_docker_hostname, get_here
+
+HERE = get_here()
+HOST = get_docker_hostname()
+PORT = '8080'
+METRICS_URL = f'http://{HOST}:{PORT}/metrics'
+
+PULSAR_VERSION = os.getenv('PULSAR_VERSION')
+
+EXPECTED_METRICS = [
```

Comment:
Some metrics in `metadata.csv` are not included in `EXPECTED_METRICS` nor `OPTIONAL_METRICS`: `pulsar.function_*`, `pulsar.source_*`, `pulsar.lb_*`, `pulsar.sink_*`, `pulsar.txn_*`, `pulsar.proxy_*`.

Reply:
Yes the others are never present in the test environment